From e4d9bf7f3f1a20c6ebee0aa61a557743d8d24af4 Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Thu, 25 Dec 2025 10:18:00 +0100
Subject: [PATCH 3/4] Build with release keys

Build with release keys instead of test-keys, help to pass safetynet

Change-Id: I07a1695995d07057431b5d892b41ce3764498298
---
 core/sysprop.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/sysprop.mk b/core/sysprop.mk
index f913d11743..9b5f46e33a 100644
--- a/core/sysprop.mk
+++ b/core/sysprop.mk
@@ -178,7 +178,7 @@ endif
 # Both of these tags will be removed and replaced with "release-keys"
 # when the target-files is signed in a post-build step.
 ifeq ($(DEFAULT_SYSTEM_DEV_CERTIFICATE),build/make/target/product/security/testkey)
-BUILD_KEYS := test-keys
+BUILD_KEYS := release-keys
 else
 BUILD_KEYS := dev-keys
 endif
-- 
2.25.1

